### ISSUE TITLE
Support for Buderus TC100.2, Bosch CT200 thermostats

### ISSUE THREAD

**Issue by emkarg** [ksya/ha-nefiteasy#61]
Hello,
Is it possible to add support for these devices?

The connection magic key issue with xmpp is solved by robertklep here https://github.com/robertklep/bosch-xmpp/issues/4.

**Comment by RobBie1221** [comment]
As far as I can see, this integration does not use bosch-xmpp? What needs to be changed? Shouldn't that be in aionefit library?

**Comment by ksya** [comment]
Yes, aionefit needs to be updated, but also this integration probably. So keeping it open for now.

**Comment by ksya** [comment]
Also there is another integration supporting both thermostats. You should be able to find it

**Comment by ksya** [comment]
It's this one https://github.com/bosch-thermostat/home-assistant-bosch-custom-component
